### PR TITLE
added: documentation & ReadMe, did some minor fixes as well

### PR DESCRIPTION
**Pull request by bhtibrewal**
- Landing page
image

- ReadMe
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,73 +4,62 @@
...
-      <a href="">
+
+      <a class="flex-align-center" href="">
         <img
           alt="artsyui-logo"
           class="artsy-logo"
           src="/src/assets/logo-v2.png"
         />
+        <h3><b>Artsy UI</b></h3>
+        <span>v1.0</span>
```
Comment (gauravsinhaweb) [inline index.html:28]: align text baseline or horizontally.

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes.

**Review by swstica — APPROVED** [review]
Looks good to me, Can be merged.
Good work:)

File: src/getting_started/color-scheme.html [hunk 2]
```diff
@@ -15,32 +15,28 @@
...
         </div>
     </header>
 
     <!-- aside start -->
     <aside class="cl-aside">
         <!-- Getting Started -->
         <h3>Getting Started</h3>
-        <a href="">Introduction</a>
-        <a class="active-tab" href="">Colors</a>
+        <a href="./introduction.html">Introduction</a>
```
Comment (Shivani-1524) [inline src/getting_started/color-scheme.html:36]: The transition on hover could be a lot faster.

File: src/components/button/button.css [hunk 3]
```diff
@@ -116,22 +120,22 @@
   background-color: var(--light-text);
   transition: 0.4s cubic-bezier(0.39, 0.575, 0.565, 1);
```
Comment (Shivani-1524) [inline src/components/button/button.css:121]: could reduce the transition speed to 0.2s for better ux

File: src/components/image/image.html [hunk 4]
```diff
@@ -67,10 +62,10 @@ <h3>Round Image</h3>
         </div>
 
         <div class="component-container">
-            <div class="artsy-img img-round ">
+            <div class="artsy-img img-round w-30">
                 <img src="https://picsum.photos/800/400" alt ="random images" loading="lazy" />
```
Comment (Shivani-1524) [inline src/components/image/image.html:66]: better text for alt tag such as component image is more descriptive for screen readers

**Pull request merged**